apps/full-site-editing/full-site-editing-plugin/global-styles/src/dom-updater.js

```diff
@@ -14,23 +15,32 @@ export default ( options, getOptionValue ) => {
 	domReady( () => {
 		const current = {};
 		const cssVariables = {};
+		const styleElement = document.createElement( 'style' );
+		document.body.appendChild( styleElement );
```

Comment:
Could we add the style element to the head, instead?

Reply:
IIRC I did that initially and that resulted in the styles being overridden by the cascade.

Reply:
Just double-checked to confirm:
The defaults for the Global Styles are set at the end of the body as well. Placing the style element in the head means the styles would be overridden by the default styles' placement in the cascade.
<img width="983" alt="Screenshot 2020-03-26 at 17 57 49" src="https://user-images.githubusercontent.com/1562646/77674145-6989d780-6f8b-11ea-9aab-8650e73719af.png">

Reply:
ah, that makes sense :+1: